tests/Resources/ti.network.httpclient.addontest.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe('Titanium.Network.HTTPClient', function () {
+	this.timeout(6e4);
+
+	it('TIMOB-27767 - trigger error callback for invalid URL', function (finish) {
+		var xhr = Ti.Network.createHTTPClient();
+		xhr.setTimeout(6e4);
+
+		xhr.onerror = function (e) {
+			finish();
+		};
```

Comment:
add `onload` handler and quickly fail the test if it gets fired?
```js
xhr.onload = e => finish(new Error('Expected onerror to be triggered for invalid URL'));
```

Reply:
@sgtcoolguy  Actual fix is in APSHTTPClient library . See PR https://github.com/appcelerator/APSHTTPClient/pull/50 .

In jira ticket there are 3 PRs.
1. https://github.com/appcelerator/APSHTTPClient/pull/50 -  Actual fix
2. https://github.com/appcelerator/aps_sdk/pull/341 - To use updated libAPSHTTPClient.a
3. https://github.com/appcelerator/titanium_mobile/pull/11513 - To use updated libraries in SDK.

Reply:
'onload' event was not fired in this case. I think there is no need to add 'onload' handler.